backend/department.py

```diff
@@ -36,7 +37,7 @@ def department():
             body = request.json
             post_dept = """INSERT INTO Department (dept_id,dept_name,dept_mgr)
             VALUES ( %s, %s, %s)"""
-            data =(body['dept_id'], body['dept_name'], body['dept_mgr'])
```

Comment:
random number can still be duplicated right. better to do auto-increment in schema